tests/kafkatest/tests/core/transactions_test.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, test_context):
         self.num_output_partitions = 3
         self.num_seed_messages = 100000
         self.transaction_size = 750
-        self.transaction_timeout = 10000
+        self.transaction_timeout = 30000
```

Comment:
If the producer can block for up to 30 seconds, do we want a slightly longer transaction timeout, just to have some buffer?

Reply:
That's fair. Maybe we can add 5-10s padding.